src/migrations.ts

```diff
@@ -16,6 +18,30 @@ export async function applyMigrations(): Promise<void> {
     await migrate({ client }, './migrations', {
       migrationTableName: 'auth.migrations',
     });
+  } catch (error: any) {
+    const userFieldsMigration = path.join(
```

Comment:
Could you add an inline comment so we know why this code is here when we look at this 3+ months from now